<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeManager</class>
 <widget class="QDialog" name="RecipeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <widget class="QListView" name="list_all_recipes"/>
   </item>
   <item row="0" column="1" rowspan="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>124</width>
       <height>348</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>123</width>
       <height>358</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="6" colspan="2">
    <widget class="QListView" name="list_user_recipes"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ADD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn_add">
     <property name="text">
      <string>&gt;&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btn_delete">
     <property name="text">
      <string>&lt;&lt;&lt;&lt;&lt;&lt;&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;DELETE&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>889</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ingredients</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QListView" name="list_ingredients"/>
   </item>
   <item row="9" column="0" colspan="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>762</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="5">
    <widget class="QPushButton" name="btn_leave">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
